calicoctl/commands/get.go

```diff
@@ -165,7 +165,7 @@ Description:
 	log.Infof("results: %+v", results)
 
 	if results.err != nil {
-		fmt.Printf("Error getting resources: %v\n", results.err)
+		fmt.Printf("Failed to execute command: %v\n", results.err)
```

Comment:
I've got two branches here now.  If the input file is invalid then it writes "Failed to execute command:  <msg about the file>"  whereas if it fails for another reason then it writes "Failed to get resources"

That'll be more consistent with the error messages for the create/apply/delete/update commands.